sdcm/fill_db_data.py

```diff
@@ -2954,10 +2961,10 @@ def fill_db_data(self):
             session.default_consistency_level = ConsistencyLevel.QUORUM
 
             session.execute("""
-                CREATE KEYSPACE IF NOT EXISTS keyspace1
+                CREATE KEYSPACE IF NOT EXISTS keyspace_fill_db_data
                 WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '3'} AND durable_writes = true;
                 """)
-            session.execute("USE keyspace1;")
+            session.execute("USE keyspace_fill_db_data;")
```

Comment:
Use `session.set_keyspace`

_Set the default keyspace for all queries made through this Session. This operation blocks until complete_
https://docs.datastax.com/en/developer/python-driver/3.19/api/cassandra/cluster/#cassandra.cluster.Session

Reply:
fixed.